keyboards/glencreag/wk84/keyboard.json

```diff
@@ -0,0 +1,286 @@
+            {"matrix": [2, 3], "x": 35, "y": 20, "flags": 4},
+            {"matrix": [2, 4], "x": 45, "y": 20, "flags": 4},
+            {"matrix": [2, 5], "x": 55, "y": 20, "flags": 4},
+            {"matrix": [2, 6], "x": 65, "y": 20, "flags": 4},
+            {"matrix": [2, 7], "x": 75, "y": 20, "flags": 4},
+            {"matrix": [2, 8], "x": 85, "y": 20, "flags": 4},
+            {"matrix": [2, 9], "x": 95, "y": 20, "flags": 4},
+            {"matrix": [2, 10], "x": 105, "y": 20, "flags": 4},
+            {"matrix": [2, 11], "x": 115, "y": 20, "flags": 4},
+            {"matrix": [2, 12], "x": 125, "y": 20, "flags": 4},
+            {"matrix": [2, 13], "x": 140, "y": 20, "flags": 4},
+            {"matrix": [1, 15], "x": 150, "y": 20, "flags": 4},
+            {"matrix": [1, 14], "x": 150, "y": 10, "flags": 4},
+            {"matrix": [1, 13], "x": 140, "y": 10, "flags": 4},
+            {"matrix": [1, 12], "x": 120, "y": 10, "flags": 4},
+            {"matrix": [1, 11], "x": 110, "y": 10, "flags": 4},
+            {"matrix": [1, 10], "x": 100, "y": 10, "flags": 4},
+            {"matrix": [1, 9], "x": 90, "y": 10, "flags": 4},
+            {"matrix": [1, 8], "x": 80, "y": 10, "flags": 4},
+            {"matrix": [1, 7], "x": 70, "y": 10, "flags": 4},
+            {"matrix": [1, 6], "x": 60, "y": 10, "flags": 4},
+            {"matrix": [1, 5], "x": 50, "y": 10, "flags": 4},
+            {"matrix": [1, 4], "x": 40, "y": 10, "flags": 4},
+            {"matrix": [1, 3], "x": 30, "y": 10, "flags": 4},
+            {"matrix": [1, 2], "x": 20, "y": 10, "flags": 4},
+            {"matrix": [1, 1], "x": 10, "y": 10, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 10, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 4},
+            {"matrix": [0, 1], "x": 10, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 20, "y": 0, "flags": 4},
+            {"matrix": [0, 3], "x": 30, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 40, "y": 0, "flags": 4},
+            {"matrix": [0, 5], "x": 50, "y": 0, "flags": 4},
+            {"matrix": [0, 6], "x": 60, "y": 0, "flags": 4},
+            {"matrix": [0, 7], "x": 70, "y": 0, "flags": 4},
+            {"matrix": [0, 8], "x": 80, "y": 0, "flags": 4},
+            {"matrix": [0, 9], "x": 90, "y": 0, "flags": 4},
+            {"matrix": [0, 10], "x": 100, "y": 0, "flags": 4},
+            {"matrix": [0, 11], "x": 110, "y": 0, "flags": 4},
+            {"matrix": [0, 12], "x": 120, "y": 0, "flags": 4},
+            {"matrix": [0, 13], "x": 130, "y": 0, "flags": 4},
+            {"matrix": [0, 14], "x": 140, "y": 0, "flags": 4},
+            {"matrix": [0, 15], "x": 150, "y": 0, "flags": 4}
+        ],
+        "react_on_keyup": true,
+        "led_count": 84,
+        "max_brightness": 160,
+        "default":{
+            "sat": 255,
+            "val": 160,
+            "speed": 195
+        },
+        "val_steps": 40,
+        "speed_steps": 60,
```

Comment:
These seem like rather large values to set as the default. Users should configure this themselves if they'd like.
```suggestion
```